### PR TITLE
create outgoing KNXIPFrame and CEMIFrame directly

### PR DESCRIPTION
**Pull request by farmio**
- initialize KNXIPBody objects directly, instead of creating empty objects
- create (outgoing) KNXIPFrame from KNXIPBody

- initialize CEMIFrame objects directly, instead of creating empty objects
- create (outgoing) CEMIFrame from Telegram

This saves a couple of function calls and is better readable imho.

### CODE REVIEW

**Comment by farmio** [comment]
If you like it we could also make CEMIFrame initializeable (staticmethod that takes a telegram and some additional information).

File: xknx/io/connect.py [hunk 1]
```diff
@@ -21,17 +21,18 @@ def __init__(self, xknx, udp_client):
         self.communication_channel = 0
         self.identifier = 0
 
-    def create_knxipframe(self):
+    def create_knxipframe(self) -> KNXIPFrame:
```
Comment (marvin-w) [inline xknx/io/connect.py:24]: This reminds that I wanted to integrate a mypy testrunner...

**Comment by marvin-w** [comment]
I like the idea!

**Comment by Julius2342** [comment]
Yes, that makes sense!

File: xknx/io/request_response.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ async def start(self):
     async def send_request(self):
         """Build knxipframe (within derived class) and send via UDP."""
         knxipframe = self.create_knxipframe()
-        knxipframe.normalize()
+        # knxipframe.normalize()
```
Comment (marvin-w) [inline xknx/io/request_response.py:44]: Typo?

**Review by marvin-w — APPROVED** [review]
Have a look at the one possible too, otherwise nice :)

**Pull request merged**